src/sonic-yang-models/yang-models/sonic-portchannel.yang

```diff
@@ -80,6 +80,15 @@ module sonic-portchannel {
                         range 1..1024;
                     }
                 }
+               
+               leaf mode {
+                   description "PortChannel Modes values will be from "routed|access|trunk. Default val is routed.";
+                   type string {
```

Comment:
We should use the new type you defined in sonic-types.yang.

Reply:
Ok will update it

Reply:
updated